modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -2,14 +2,36 @@
 
 module AccreditedRepresentativePortal
   class PowerOfAttorneyRequest < ApplicationRecord
+    module ClaimantTypes
```

Comment:
If you're adding a namespace/module here, you'll need to reflect that in the filename/directory structure in order for the Zeitwerk autoloading to work properly

Reply:
Thanks Lindsey. I think this will get loaded in when the containing model gets evaluated which itself follows the required structure.

Reply:
ah so I def overlooked that... you're right, zeitwerk won't load that separately!